webextensions/native-messaging-host/wix.json

```
@@ -25,8 +25,8 @@
   },
   "choco": {
     "description": "Native Messaging Host for FlexConfirmMail",
-    "project-url": "https://github.com/clear-code/flex-confirm-mail",
+    "project-url": "https://github.com/clear-code/flex-confirm-mail-stable",
     "tags": "thunderbird addon",
-    "license-url": "https://github.com/clear-code/flex-confirm-mail/blob/master/webextensions/native-messaging-host/LICENSE"
+    "license-url": "https://github.com/clear-code/flex-confirm-mail-stable/blob/master/webextensions/native-messaging-host/LICENSE"
```

```suggestion
    "license-url": "https://github.com/FlexConfirmMail/Thunderbird/tree/stable/webextensions/native-messaging-host/LICENSE"
```

https://github.com/clear-code/flex-confirm-mail-stable は存在しないURLなので、上記のようにstableブランチを指定するのが良さそうに思いました。

---

こちらも同様でした。